templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1> Delete car</h1>
+  <p>Are you sure want to delete the car: {{ car }}?</p>
+  <p><i>All Manufacturers with this car will be also deleted</i></p>
+
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-danger">
```

Comment:
From checklist:
Add Cancel button apart from Delete one. The Cancel button will lead to the previous page the user was on.